src/client/components/MenuPage.jsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import { useAppDispatch, useAppSelector } from '../state/hooks';
+import { Button } from 'react-bulma-components';
+import allActions from '../state/actions/allActions';
+import styled from 'styled-components';
+import MenuItemContainer from './MenuItemContainer.jsx';
+import {OrangeButton} from '../styles/shared.tsx';
+import { useHistory } from 'react-router-dom';
+
+const MainConatiner = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+`;
+
+const CheckoutButton = styled(OrangeButton)`
+  border-radius: 7px;
+  width: 50%;
+`;
+
+export default function MenuPage () {
+  const currentItem = useAppSelector((state)=>state.currentMenuItem)
+  const totalOrdersPrice = useAppSelector((state)=>state.allOrderItems.ordersTotal)
+  const dispatch = useAppDispatch();
+  const history = useHistory();
+
+  function clickHandler (entry) {
+    //reroute
+    console.log(entry.name)
+    console.log(entry.price)
+    dispatch(allActions.UpdateItemName(entry.name));
+    dispatch(allActions.UpdateItemPrice(entry.price));
+    dispatch(allActions.UpdateItemDescription(entry.description));
```

Comment:
would highly recommend destructuring the individual actions from the allActions to keep code dry!